### ISSUE TITLE
Possible race condition bug

### ISSUE THREAD

**Issue by jfairbank** [jfairbank/run-elm#1]
Occasionally seeing this error pop up when using `run-elm` with `nodemon`:

```
Error: Compilation failed
elm-make: /my/user/folder/RunElmMain.elm: openFile: does not exist (No such file or directory)
```

**Comment by jfairbank** [comment]
Another error:

```
Error: Compilation failed
elm-make: /my/user/folder/RunElmMain.elm: openFile: does not exist (No such file or directory)
elm-make: thread blocked indefinitely in an MVar operation
```

**Comment by kachkaev** [comment]
I may also be hitting this when `run-elm` is called to frequently. Perhaps the second call deletes `RunElmMain.elm` that is still being used by the first call. Could calling the temp file something like `RunElmMain${TIMESTAMP}.elm` solve the issue?

**Comment by jfairbank** [comment]
@kachkaev Possibly. I haven't had time to debug this unfortunately. I'd gladly welcome a PR for naming the file with a timestamp or guid of some sort, though.

**Comment by kachkaev** [comment]
Can we consider this as closed because of #5 and come back to the discussion if the problem occurs again?

**Comment by jfairbank** [comment]
Sure, sounds good to me.
